bin/notification.py

```diff
@@ -31,8 +31,13 @@ def handle_batch_event(self, event):
 
     def handle_single_event(self, body):
         query = AthenaQuery(**json.loads(body))
-        if query.data_scanned > self.config.SLACK_ALERT_DATA_THRESHOLD:
-            self.send_slack_notification(query)
+        # alert may be sent either to the user who submitted the query or to the admin team channel, or to both
+        # here we decide where the notification is to be sent
+        # if at least one threshold is reached, we call the send function with params where the message is to be sent
+        is_send_to_user = query.data_scanned > self.config.SLACK_ALERT_DATA_USER_THRESHOLD
+        is_send_to_admin_channel = query.data_scanned > self.config.SLACK_ALERT_DATA_CHANNEL_THRESHOLD
+        if is_send_to_user or is_send_to_admin_channel:
```

Comment:
why do you check this condition here and again in the send_slack_notification function?